design/specs/add-formula-column.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Listing Views Spec
+description: 
+published: true
+date: 2022-06-20T08:24:04Z
+tags: 
+editor: markdown
+dateCreated: 2022-06-20T08:24:04Z
+---
+
+## Context
+
+In addition to data columns, we have formula columns as well. These columns are basically derived from a formula. A formula may or may not use another data column to generate the formula column. The formula can also use parameters of different data types other than columns. 
+
```

Comment:
Let us also clarify that the use of formulas is currently restricted to queries created with Data Explorer. Let's also include a link to the wiki section that goes into greater detail about formulas.

Reply:
So just confirming again, users will only be allowed to create formula columns while exploring the query in the data explorer, right?

Will surely add the link.

Reply:
For now we'll only allow formula columns while building queries (Data Explorer). In the future we might use formulas in other contexts.